drivers/SmartThings/zigbee-contact/src/test/test_zigbee_contact.lua

```diff
@@ -31,7 +31,7 @@ local mock_device = test.mock_device.build_test_zigbee_device(
 zigbee_test_utils.prepare_zigbee_env_info()
 local function test_init()
   test.mock_device.add_test_device(mock_device)
-  zigbee_test_utils.init_noop_health_check_timer()
+  --zigbee_test_utils.init_noop_health_check_timer() removal due to change to stop health checks going forward
```

Comment:
Since the health check disable is happening in the base driver, we will need to remove this test config from all of the tests in the `test/` folder, not just this test.  The same for the `motion` driver.